files/ohpc_install.yml

```diff
@@ -117,15 +141,11 @@
                                 kargs=''
                                 additional_modules='qcom_emac'
                                 enable_opensm=True
-                                enable_ipoib=True
-                                enable_ifup=True
-                                enable_linux_ib=False
-                                enable_mellanox_ib=True
                                 mofed_install_opts="--distro rhel7.5alternate"
                                 enable_socket_direct=True
                         elif [ ${node} == 'x86ohpc' ]; then
-                                mellanox_iso_url=""
-                                mellanox_iso_file=""
+                                mellanox_iso_url="http://content.mellanox.com/ofed/MLNX_OFED-4.5-1.0.1.0/MLNX_OFED_LINUX-4.5-1.0.1.0-rhel7.6-x86_64.iso"
```

Comment:
We better cache that binary, too. Can you use the cacher script here?

Reply:
I was thinking for maintenance reasons. In the future, only changing the URL would be necessary as a patch to upgrade the driver, instead of caching by hand every time. But it's not critical.